src/Notify/Resources/doc/index.rst

```diff
@@ -0,0 +1,140 @@
+Symfony UX Notify
+===================
+
+Symfony UX Notify is a Symfony bundle providing light dropzones for
+file inputs in Symfony Forms. It is part of `the Symfony UX initiative`_.
+
+Symfony UX Notify is a Symfony bundle integrating realtime native
+notifications in Symfony applications using `Mercure`_.
+It is part of `the Symfony UX initiative`_.
+
+Installation
+------------
+
+Before you start, make sure you have `Symfony UX configured in your app`_.
+
+Then, install this bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-notify
+
+    # Don't forget to install the JavaScript dependencies as well and compile
+    $ yarn install --force
+    $ yarn encore dev
+
+Also make sure you have at least version 3.0 of
+`@symfony/stimulus-bridge`_ in your ``package.json`` file.
+
+Configuration
+-------------
```

Comment:
I'd move this to the last section, as most people will not actually need to configure this. And if you DO have multiple hubs, you will be looking specifically for this config.